Gopkg.lock

```diff
@@ -1,6 +1,31 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  branch = "parse-constraints-with-dash-in-pre"
+  name = "github.com/Masterminds/semver"
+  packages = ["."]
+  revision = "a93e51b5a57ef416dac8bb02d11407b6f55d8929"
+  source = "https://github.com/carolynvs/semver.git"
```

Comment:
These are all dependencies of the `dep` tool, which must be vendored as well. In a followup PR, I'm going to `dep prune` away the bloat in `vendor/`).